fineract-provider/src/main/java/org/apache/fineract/infrastructure/campaigns/email/exception/EmailConfigurationNotFoundException.java

```diff
@@ -28,4 +29,9 @@
     public EmailConfigurationNotFoundException(final String name) {
```

Comment:
Just curious but wouldn't it be better to add the 3rd param here instead of duplicating the method?